ge_releaser/cmd/prep.py

```diff
@@ -131,32 +79,35 @@ def _update_getting_started_snippet(release_version: str) -> None:
 
     If the .mdx file already exists, it is overwritten when the script runs.
     """
-    with open(GETTING_STARTED_VERSION, "w") as snippet_file:
+    with open(GxFile.GETTING_STARTED_VERSION, "w") as snippet_file:
         lines = ("```\n", f"great_expectations, version {release_version}", "\n```")
         snippet_file.writelines(lines)
 
 
 def _update_changelogs(
-    github_org: Organization,
     github_repo: Repository,
-    current_version: str,
+    last_version: str,
     release_version: str,
 ) -> None:
     relevant_prs: List[PullRequest] = _collect_prs_since_last_release(
-        github_repo, current_version
+        github_repo, last_version
     )
 
-    changelog_entry: ChangelogEntry = ChangelogEntry(github_org, relevant_prs)
+    changelog_entry: ChangelogEntry = ChangelogEntry(relevant_prs)
 
-    changelog_entry.write(CHANGELOG_MD, current_version, release_version)
-    changelog_entry.write(CHANGELOG_RST, current_version, release_version)
+    changelog_entry.write(GxFile.CHANGELOG_MD, last_version, release_version)
+    changelog_entry.write(GxFile.CHANGELOG_RST, last_version, release_version)
 
 
 def _collect_prs_since_last_release(
     github_repo: Repository,
-    current_version: str,
+    last_version: str,
 ) -> List[PullRequest]:
-    last_release: dt.datetime = github_repo.get_release(current_version).created_at
+    # 20220923 - Chetan - Currently, this grabs all PRs from the last release until the moment of program execution.
```

Comment:
The tag is usually done close to the time of release, so hopefully this won't be an issue.